dropwizard-parent-pom/pom.xml

```diff
@@ -31,17 +31,16 @@
       idea is that these dependencies should only ever change versions when we upgrade to a
       newer version of dropwizard.
     -->
-    <dropwizard.api.key.bundle.version>0.8.5</dropwizard.api.key.bundle.version>
     <dropwizard.configurable.assets.bundle.version>0.8.5</dropwizard.configurable.assets.bundle.version>
     <dropwizard.metrics.datadog.ecs.version>0.2.0</dropwizard.metrics.datadog.ecs.version>
     <dropwizard.redirect.bundle.version>0.8.5</dropwizard.redirect.bundle.version>
```

Comment:
These bundles need to be upgraded to 1.0.0 versions (they've already been released)